### PR TITLE
Add KEG_DB_ENGINE_OPTIONS Configuration

### PR DESCRIPTION
**Pull request by nZac**
#59 should be done at some point, however I didn't have time for it right now.'

This removes [all of this from the cookie cutter](https://github.com/level12/keg-app-cookiecutter/blob/master/%7B%7Bcookiecutter.project_dashed%7D%7D-src/%7B%7Bcookiecutter.project_namespace%7D%7D/app.py#L42-L71)

### CODE REVIEW

**Comment by guruofgentoo** [comment]
The connection between what is being removed from the cookie-cutter and what is here is unclear:
- the cookie-cutter updates are being set in apply_pool_defaults
- the new setup is options to pass to create_engine
Are both setting options that eventually get to create_engine? If so, then this should be fine.

**Comment by nZac** [comment]
Sorry, didn't have the commit up there yet: https://github.com/level12/keg-app-cookiecutter/pull/62/commits/e15cea8d22501931d6b8c51a288e1ac531adbc4d

File: keg/db/__init__.py [hunk 1]
```diff
@@ -8,6 +8,19 @@
...
+        """Override some driver specific settings"""
+        super(KegSQLAlchemy, self).apply_driver_hacks(app, info, options)
+
+        engine_opts = app.config.get('KEG_DB_ENGINE_OPTIONS', {})
+
+        # Turn on SA pessimistic disconnect handling by default:
+        # http://docs.sqlalchemy.org/en/latest/core/pooling.html#disconnect-handling-pessimistic
+        engine_opts.setdefault('pool_pre_ping', True)
+
+        if engine_opts:
```
Comment (rsyring) [inline keg/db/__init__.py]: by using setdefault() just above, this `if` is not necessary.

Comment (nZac) [reply keg/db/__init__.py]: Thanks, fixed.

**Pull request merged**